packages/core/src/components/button/button-group.md

```diff
@@ -22,14 +22,29 @@ visual style, it’s recommended to apply the same `intent` to all buttons withi
 
 @reactCodeExample ButtonGroupIntentExample
 
+@## Variant
+
+Most of **ButtonGroup**'s props are also supported by [**Button**](#core/components/buttons) directly. Setting these props on **ButtonGroup** will apply the same value to all buttons in the group. Note that most modifiers, once enabled on the group, cannot be overridden on child buttons (due to the cascading nature of CSS).
```

Comment:
This bit feels a bit incongruous in this position, which is refering to a specific prop. Its also duplicated in the outlined and minimal section below. I think we should take this section and move it to the top of the page and remove it from both of these sections.

Reply:
d1befe6fc3d2477170d238b832cc4ac184290dcd